### ISSUE TITLE
[CoE Starter Kit - QUESTION] does coe has any feature where it can help duplicate existing DLP policy?

### ISSUE THREAD

**Issue by manuelap-msft** [microsoft/coe-starter-kit#5845]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

does coe has any feature where it can help duplicate existing DLP policy?

### What solution are you experiencing the issue with?

None

### What method are you using to get inventory and telemetry?

None

**Comment by manuelap-msft** [comment]
Yep there is in the DLP Editor v2 if you select the "copy" icon in front of the arrow
![image](https://github.com/microsoft/coe-starter-kit/assets/55705669/156037f9-17d4-426a-825b-854ef363a7f3)

Note that there's an open issue that this copies only the connectors, and if you have [granular connector configuration](https://learn.microsoft.com/en-us/power-platform/admin/dlp-granular-controls) that is lost on copy.

**Comment by sanjeev40084** [comment]
@manuelap-msft thanks that helps. I think i found a bug will be raising new issue for that.

**Comment by sanjeev40084** [comment]
looks like you have already fixed the bug here: #5823
